app/presenters/check_answers/letters_calls_card.rb

```diff
@@ -0,0 +1,17 @@
+module CheckAnswers
+  class LettersCallsCard < Base
```

Comment:
this should try and leverage what was done for the summary if possible

Reply:
I was initially going to do this, but so much of it is different that I thought to separate it out

Cost summary version: 
![image](https://github.com/ministryofjustice/laa-claim-non-standard-magistrate-fee-backend/assets/26544538/6dfdea23-c30f-408e-b7d3-3c52958092aa)
No summation in the title, an overarching heading, a totals tow, heading column text different

Check answers version: 
<img width="409" alt="image" src="https://github.com/ministryofjustice/laa-claim-non-standard-magistrate-fee-backend/assets/26544538/d6c5f79d-2fd6-4c1b-a238-f5a52c58c9b3">
We are showing the number of each and not the cost of each